internal/source/dispatcher.go

```diff
@@ -0,0 +1,88 @@
+package source
```

Comment:
I just split the Dispatcher service into two:
- `Scheduler` analyzes the provided configuration and based on that schedules plugin sources.
- and `Dispatcher` provides functionality to starts a given plugin, watches for incoming events and calling all notifiers to dispatch received event.
-